### ISSUE TITLE
The Browser or App may not be secured . I got this error in google phishlet, i think google again updated their policy ,any updates related to this issue?

### ISSUE THREAD

**Issue by WizAdward** [kgretzky/evilginx2#1094]
![img](https://github.com/user-attachments/assets/1788e10b-877d-45f2-921f-edc2831078dc)

**Comment by rickyrik001** [comment]
hey @WizAdward ,To Bypass this issue you have to tweak Post Request,you can use EvilPuppet or any other tool that uses Puppeteer in the background to capture and stream HTML content to a target browser.
if you need help contact me  
telegram @Ricky_rik1
rickyrik at protonmail.com

**Comment by simplerhacking** [comment]
**Google phishlets work perfectly fine.** 

You do not need anything fancy or evilpuppet. You can even just extract the necessary data via custom js per the official documentation, that's why Kuba put it there. There is no need to use all of extra tools or even the pro version. Google rolled out the new v3 login, I highly doubt they patch it or change it anytime soon. (Look at v3 release notes & recaptcha notes).

https://cloud.google.com/recaptcha/docs/release-notes 

Google phishlets for Evilginx 3.0 work fine. Google sign-in redirects instantly to myaccount as normal, even switching apps will not prompt re-authentication via the main account. 

![Google Phishlet Fine Update](https://github.com/user-attachments/assets/99409ffa-2f5c-455f-8f22-c33430933d82)
![Google Phishlet Fine Update 2](https://github.com/user-attachments/assets/04c5256b-a254-4c98-9bc1-d2ade2f54048)
![Google Phishlet Fine Update 3](https://github.com/user-attachments/assets/9521110f-a9b6-4363-ac18-72511c38706e)
![Google Phishlet Fine Update 4](https://github.com/user-attachments/assets/80d6f870-4c8f-4064-9b66-11112a2aa117)

**Comment by rickyrik001** [comment]
@simplerhacking yes it works fine but for some accounts ,but for the most of the account it shows this error ,google soon patch all accounts as it does previously if you remember..

**Comment by bigherocenter** [comment]
> @simplerhacking yes it works fine but for some accounts ,but for the most of the account it shows this error ,google soon patch all accounts as they did previously , if you remember.

It mean that each account work different  to login?
And also it show error for some  country.

**Comment by antonbek89** [comment]
Hey

Do you solve the issue?

@simplerhacking  can you please explain the full process?

**Comment by simplerhacking** [comment]
@antonbek89 I might record a long tutorial on the Google Phishlet process that I mentioned for the v3 update. Everything still works fine even 5 months after I replied to this thread. Specifically for this, the bulk is JS and DOM inject/manipulation. It will just take a while to film and narrate such a tutorial. Reverse-engineering this just takes a while and it takes even longer to explain coherently to an audience so you can understand what I'm actually talking about without getting lost. 

I may make a long form video tutorial in the future on my site or the research blog.

**Comment by bradapele** [comment]
@simplerhacking Can you give me the google phishlet you used?

**Comment by Hosted-EU** [comment]
👍✅ https://youtu.be/Ifuj1S9kTFI?si=PcAmtWjjmZM3zaSN

**Comment by Hosted-EU** [comment]
Join new Evilginx Group Chat 🔥https://t.me/evilginx_group
